src/app.js

```diff
@@ -1 +1,23 @@
 // write code here
+const fs = require('fs');
+const path = require('path');
+
+function moveFile() {
+  const [from, to] = process.argv.slice(2);
+
+  const dest =
```

Comment:
The `dest` variable is not declared. You need to declare it using `let` or `const` before assigning a value to it.